mypy/build.py

```diff
@@ -792,6 +792,21 @@ def find() -> Optional[str]:
                 dir = os.path.normpath(os.path.join(pathitem, dir_chain))
                 if os.path.isdir(dir):
                     dirs.append(dir)
+            try:
+                user_dir = site.getusersitepackages()
```

Comment:
Doesn't this use the site packages of the mypy installation, not that of the code being typechecked?

Reply:
Yes I need to add that. Thanks for the reminder!